framework/test/jest/unit/specs/controller/__snapshots__/application.spec.js.snap

```diff
@@ -20,7 +20,7 @@ Object {
       },
     },
     "ipc": Object {
-      "enabled": true,
+      "enabled": false,
```

Comment:
Was it intentional to make it `false`? I thought for `devnet` IPC is `true` by default